<launch>
	<arg name="map_file" default="$(find e190_bot)/params/maps/map_demo.yaml"/>
	<arg name="rviz_cfg" default="-d $(find e190_bot)/params/rviz/prm_rviz.rviz" />


	<node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" >
    <param name="frame_id" type="string" value="/base_link"/> 
	</node>

	<node name="prm_planning" pkg="e190_bot" type="prm_planning.py" output="screen"/>

	<node pkg="rviz" type="rviz" name="rviz" args="$(arg rviz_cfg)"/>

    <node name="control" pkg="e190_bot" type="control.py"/>

    <!-- <node name="particleFilter" pkg="e190_bot" type="particleFilter.py"/> -->

    <node name="pointTracking" pkg="e190_bot" type="pointTracking.py" output="screen"/>

</launch>